### PR TITLE
RIP-528 Fix duplicate mailing list route config

### PR DESCRIPTION
**Pull request by pauline2k**
https://jira-secure.berkeley.edu/browse/RIP-528

### CODE REVIEW

File: src/router.ts [hunk 1]
```diff
@@ -111,18 +111,13 @@ const routes:RouteRecordRaw[] = [
       {
         component: MailingListCreate,
         name: 'Create Mailing List',
-        path: '/mailing_list/create'
+        path: '/mailing_list/create/:canvasSiteId?'
```
Comment (johncrossman) [inline src/router.ts:114]: @pauline2k - did you intend the path to end with "?"

Comment (pauline2k) [reply src/router.ts:114]: @johncrossman Yes, the "?" makes it optional. This seems to be the way vue-router wants it defined, as opposed to having two separate routes with and without the param.

**Comment by pauline2k** [comment]
The Travis failure is an unrelated new linter snit; created https://jira-secure.berkeley.edu/browse/RIP-566 to address.

**Comment by johncrossman** [comment]
@pauline2k - does this PR need a rebase to lose the shadowing error?

**Comment by johncrossman** [comment]
Ah, sorry. I missed your comment. Merging.

**Pull request merged**